packages/block-library/src/gallery/edit.js

```diff
@@ -302,14 +302,16 @@ function GalleryEdit( props ) {
 	}, [ isSelected ] );
 
 	useEffect( () => {
-		// linkTo attribute must be saved so blocks don't break when changing
-		// image_default_link_type in options.php
-		if ( ! linkTo ) {
```

Comment:
minor nitpick: `Platform.OS === 'web'` could be added here instead of below, to keep the code indentation the same and make the diff here smaller.

Reply:
Something like `if ( Platform.OS === 'web' && ! linkTo ) {`

Reply:
This is a good suggestion. Thanks. I will resolve this tomorrow when making the respective updates.

Reply:
Resolved here [5e80ad1](https://github.com/WordPress/gutenberg/pull/26227/commits/5e80ad1d930601f708a82dc1f2faf57d5166063e)